src/frontend/enzymes.jl

```diff
@@ -68,12 +71,12 @@ that limits the total sum of the listed genes to the given limit.
 """
 function enzyme_constrained_flux_balance_constraints(
     model::A.AbstractFBCModel;
-    reaction_isozymes::Dict{String,Dict{String,Isozyme}},
+    reaction_isozymes::Dict{String,Dict{String,IsozymeT{T1}}},
```

Comment:
you don't use the T1 type anywhere so you shouldn't need to capture it here. (Or you can write `IsoT{<:Real}` right in place.)